tests/integration/conftest.py

```diff
@@ -27,22 +27,22 @@
 # ======================================================================================================================
 # Globals
 # ======================================================================================================================
-DEFAULT_ASC_GLOBAL_PROPERTIES = {"BUILD_URL": "BUILD_URL",
-                                 "BUILD_NUMBER": "BUILD_NUMBER",
-                                 "RE_JOB_ACTION": "RE_JOB_ACTION",
-                                 "RE_JOB_IMAGE": "RE_JOB_IMAGE",
-                                 "RE_JOB_SCENARIO": "RE_JOB_SCENARIO",
+DEFAULT_ASC_GLOBAL_PROPERTIES = {"BUILD_URL": "https://rpc.jenkins.cit.rackspace.net/job/PM_rpc-openstack-pike-rc-xenial_mnaio_no_artifacts-swift-system/78/",  # noqa
+                                 "BUILD_NUMBER": "78",
+                                 "RE_JOB_ACTION": "system",
+                                 "RE_JOB_IMAGE": "xenial_mnaio_no_artifacts",
+                                 "RE_JOB_SCENARIO": "swift",
                                  "RE_JOB_BRANCH": "pike-rc",
-                                 "RPC_RELEASE": "RPC_RELEASE",
-                                 "RPC_PRODUCT_RELEASE": "RPC_PRODUCT_RELEASE",
-                                 "OS_ARTIFACT_SHA": "OS_ARTIFACT_SHA",
-                                 "PYTHON_ARTIFACT_SHA": "PYTHON_ARTIFACT_SHA",
-                                 "APT_ARTIFACT_SHA": "APT_ARTIFACT_SHA",
+                                 "RPC_RELEASE": "r16.2.0",
+                                 "RPC_PRODUCT_RELEASE": "pike",
+                                 "OS_ARTIFACT_SHA": "Unknown",
+                                 "PYTHON_ARTIFACT_SHA": "Unknown",
+                                 "APT_ARTIFACT_SHA": "Unknown",
                                  "REPO_URL": "https://github.com/rcbops/rpc-openstack",
-                                 "JOB_NAME": "JOB_NAME",
-                                 "MOLECULE_TEST_REPO": "MOLECULE_TEST_REPO",
-                                 "MOLECULE_SCENARIO_NAME": "MOLECULE_SCENARIO_NAME",
-                                 "MOLECULE_GIT_COMMIT": "Unknown",
+                                 "JOB_NAME": "PM_rpc-openstack-pike-rc-xenial_mnaio_no_artifacts-swift-system",
+                                 "MOLECULE_TEST_REPO": "molecule-validate-neutron-deploy",
+                                 "MOLECULE_SCENARIO_NAME": "default",
+                                 "MOLECULE_GIT_COMMIT": "34a371af807beced059298784f71da9a89ab8639",
                                  "ci-environment": "asc"}
```

Comment:
Altering the defaults is probably a bad idea as it affects all test cases. In hindsight we probably shouldn't have put non-test values in for "REPO_URL" or "RE_JOB_BRANCH".

Instead of overriding global default properties I suggest using custom fixtures that just overwrite the global properties to fit the needs of the test case. [Check out the fixtures for the "test_github_linking" test suites for an example.](https://github.com/rcbops/zigzag/blob/master/tests/integration/test_github_linking.py#L45-L47)

Reply:
My reasoning for changing the defaults is that there will never be a scenario where the contents of RE_JOB_BRANCH is RE_JOB_BRANCH.  

Since the behavior of the tool is determined by the contents of the properties not just the presence I think that we are missing code pathways by not providing real values in all scenarios. 

My point is that the concept of defaults is no longer useful.